src/openms/source/ANALYSIS/MAPMATCHING/TransformationDescription.cpp

```diff
@@ -184,4 +184,68 @@ namespace OpenMS
     }
   }
 
+  void TransformationDescription::getDeviations(vector<double>& diffs, 
```

Comment:
I see no reason why it should not be optimized away on the compilers we support. But even if not I would still go for the cleaner version.